vacasa/connect/connect.py

```diff
@@ -1277,7 +1279,10 @@ def update_contact_finances_payload(self, contact_id, params: dict):
         """
 
         url = f"{self.endpoint}/v1/contacts/{contact_id}/finances"
-        return self._patch(url, json={'data': {'attributes': params}}, headers=self._headers()).json()
+        try:
+            r = self._patch(url, json={'data': {'attributes': params}}, headers=self._headers())
+        except HTTPError as e:
+            return e.response.json()
```

Comment:
I think we still might need to handle the case where it's not jsonifyable (e.g. a 504)